editor/src/components/canvas/controls/select-mode/cursor-overlay.tsx

```diff
@@ -19,10 +18,9 @@ export function getCursorForOverlay(editorState: EditorState): CSSCursor | null
 }
 
 export const CursorOverlay = React.memo(() => {
-  const strategyCursor = useDelayedStrategyCursor()
-  const cursor = useEditorState((store) => {
-    return Utils.defaultIfNull(strategyCursor, getCursorFromDragState(store.editor))
-  }, 'CursorOverlay cursor')
+  const cursor = useDelayedEditorState((store) => {
+    return getCursorForOverlay(store.editor)
```

Comment:
This previously meant that the canvas overlay wouldn't show the "not permitted" cursor for disappearing elements (though the test for this was returning that cursor!)

Reply:
dod this change the other delayed curso behaviors though?

Reply:
Nah I checked this. It was previously using the exact same logic, but without updating the cursor for missing dragged items